client/beefy/src/worker.rs

```diff
@@ -783,6 +770,29 @@ where
 		Ok(())
 	}
 
+	fn process_new_state(&mut self) {
+		// Handle pending justifications and/or votes for now GRANDPA finalized blocks.
+		if let Err(err) = self.try_pending_justif_and_votes() {
+			debug!(target: "beefy", "🥩 {}", err);
+		}
+
+		// Don't bother voting or requesting justifications during major sync.
+		if !self.network.is_major_syncing() {
+			// There were external events, 'state' is changed, author a vote if needed/possible.
+			if let Err(err) = self.try_to_vote() {
+				debug!(target: "beefy", "🥩 {}", err);
+			}
+			// If the current target is a mandatory block,
+			// make sure there's also an on-demand justification request out for it.
+			if let Some(block) = self.voting_oracle().mandatory_pending() {
+				// This only starts new request if there isn't already an active one.
+				self.on_demand_justifications.request(block);
+			}
+		} else {
+			info!(target: "beefy", "🥩 Skipping voting while major syncing.");
```

Comment:
Does this really needs to be info? Will this not spam the logs? Before it was also just debug.

Reply:
I don't think it's very spammy since finality notifications come in large batches during major sync, but even so, you're right, `debug` is better here, it's not exactly "useful" info to see these unless you're debugging, so yeah, I'll change it back.